packages/core/src/runtime/runtime.ts

```diff
@@ -43,6 +43,12 @@ export abstract class Runtime<TRoute = unknown, TNavigationItem = unknown> {
         this._eventBus = new EventBus({ logger: this._logger });
         this._services = services;
         this._sessionAccessor = sessionAccessor;
+
+        this._plugins.forEach(x => {
```

Comment:
i feel like i've never seen you use x as a variable name. Rename to plugin?

Reply:
😆 This is a name I used often when I don't feel like naming the argument help better understand the code.

![image](https://github.com/gsoft-inc/wl-squide/assets/794579/2cfe468a-da3c-4467-92da-6c01e946b353)